src/style.css

```diff
@@ -0,0 +1,46 @@
+.page__body {
+  margin: 0;
+}
+
+.page__block {
+  width: 100px;
+}
+
+.block {
+  display: flex;
+  flex-direction: column;
+}
+
+.block__stars {
+  padding-right: 4px;
+}
+
+.stars {
```

Comment:
move each BEM block to the separate file

Reply:
Moved.